src/pybind/mgr/mgr_util.py

```diff
@@ -73,6 +73,22 @@ def run(self):
             logger.error("task exception: %s", e)
             raise
 
+@contextlib.contextmanager
+def lock_timeout_log(lock: Lock, timeout: int = 5) -> ContextManager[bool]:
```

Comment:
Paging Python guru @sebastian-philipp, can you help me get the right syntax to resolve this tox failure? :smile:

````
mgr_util.py: note: In function "lock_timeout_log":
mgr_util.py:76: error: The return type of a generator function should be "Generator" or one of its supertypes
mgr_util.py: note: At top level:
mgr_util.py:76: error: Argument 1 to "contextmanager" has incompatible type "Callable[[Lock, int], ContextManager[bool]]"; expected "Callable[..., Iterator[<nothing>]]"
Found 2 errors in 1 file (checked 27 source files)
````

Reply:
@batrick just came across your comment.

```py
def lock_timeout_log(lock: Lock, timeout: int = 5) -> Iterator[bool]:
```

should be a solution. as the decorated function returns an iterator.

Reply:
Thanks @tchaikov!